support-frontend/assets/components/subscriptionCheckouts/address/addressFields.tsx

```diff
@@ -149,7 +149,7 @@ export function AddressFields({ scope, ...props }: PropTypes): JSX.Element {
 				id={`${scope}-stateProvince`}
 				data-qm-masking="blocklist"
 				label={props.country === 'CA' ? 'Province/Territory' : 'State'}
-				value={props.state ?? ''}
+				value={props.state}
```

Comment:
These changes are just because the Zod schema means these fields are no longer nullable, and ESLint was yelling.